pkgs/development/tools/parsing/tree-sitter/grammar.nix

```
@@ -46,6 +46,9 @@ stdenv.mkDerivation ({
       $CC -fPIC -c src/scanner.c -o scanner.o $CFLAGS
     fi
     $CC -fPIC -c src/parser.c -o parser.o $CFLAGS
+    if [[ -e parser ]]; then
+      rm -r parser
+    fi
```

```suggestion
    rm -rf parser
```
this would functionally be the same